Auth0.Windows/Auth0Client.cs

```diff
@@ -199,10 +205,20 @@ public Task<JObject> GetDelegationToken(string targetClientId, IDictionary<strin
         /// <summary>
         /// Log a user out of a Auth0 application.
         /// </summary>
-        public void Logout()
+        /// <param name="forceClearCache">Force clearing the browser cache, no matter the login method</param>
+        public void Logout(bool forceClearCache = false)

```

Comment:
@rizman Why is the `forceClearCache` parameter necessary? You are only ever calling it with the value of the `clearCache` field, so this seems a bit unnecessary...?